transformation/ccda-to-fhirr4-service/tests/service_test.bal

```diff
@@ -23,23 +23,110 @@ map<string> cdaDocumentMap = {};
 
 @test:BeforeSuite
 function beforeSuiteFunc() returns error? {
-    string patientDocument = check io:fileReadString("tests/test_patient_ccda_document.xml");
+    string validDocument = check io:fileReadString("tests/test_valid_ccda_document.xml");
     string invalidDocument = check io:fileReadString("tests/test_invalid_ccda_document.xml");
-    cdaDocumentMap["patient"] = patientDocument;
+    cdaDocumentMap["valid"] = validDocument;
     cdaDocumentMap["invalid"] = invalidDocument;
 }
 
 @test:Config {}
 function testCcdaDocumentToFhir() returns error? {
-    http:Response|error response = testClient->/transform.post(cdaDocumentMap["patient"]);
+    http:Response|error response = testClient->/transform.post(cdaDocumentMap["valid"]);
     test:assertTrue(response is http:Response, "Error occurred while transforming CCDA document to FHIR!");
     if (response is http:Response) {
         test:assertEquals(response.statusCode, 200, "Response status code mismatched!");
         json jsonPayload = check response.getJsonPayload();
         test:assertEquals(jsonPayload.resourceType, "Bundle", "Error occurred while transforming CCDA document to FHIR!");
+        
         json[] entries = <json[]>check jsonPayload.entry;
-        test:assertEquals(entries.length(), 2, "Incorrect number of bundle entries from the conversion!");
+        test:assertEquals(entries.length(), 7, "Incorrect number of bundle entries from the conversion!");
         test:assertEquals(check entries[0].'resource.resourceType, "Patient", "Incorrect resource type from the conversion!");
+
+        json[] addresses = <json[]>check entries[0].'resource.address;
+        test:assertEquals(addresses.length(), 2, "Incorrect number of addresses from the conversion!");
```

Comment:
Instead of all these assertions, can't we construct json check equality?

Reply:
Fixed in https://github.com/wso2/open-healthcare-prebuilt-services/commit/6ba8462e30e7915f54f620e1f211b28efb5d6e55